pkg/key/app.go

```diff
@@ -142,6 +150,10 @@ func IsManagedByFlux(customResource v1alpha1.App, projectName string) bool {
 	return customResource.Labels[label.ManagedBy] == "flux"
 }
 
+func IsManagedInOrg(customResource v1alpha1.App) bool {
```

Comment:
```suggestion
func IsInOrgNamespace(customResource v1alpha1.App) bool {
```

For me this name is a bit clearer. WDYT?

Reply:
I like it, it's more direct 😄